integration_tests/src/main/python/arithmetic_ops_test.py

```diff
@@ -824,6 +824,12 @@ def test_bit_not(data_gen):
     assert_gpu_and_cpu_are_equal_collect(
             lambda spark : unary_op_df(spark, data_gen).selectExpr('~a'))
 
+
+@pytest.mark.parametrize('data_gen', integral_gens + boolean_gens, ids=idfn)
+def test_bit_count(data_gen):
+    assert_gpu_and_cpu_are_equal_collect(
+        lambda spark: unary_op_df(spark, data_gen, length=10).selectExpr('bit_count(a)'))
```

Comment:
Why only 10 values?

Reply:
Sorry, that is for debugging and I forgot to clean up.